doc/api/util.md

```diff
@@ -364,16 +364,19 @@ util.formatWithOptions({ colors: true }, 'See object %O', { foo: 42 });
 // when printed to a terminal.
 ```
 
-## `util.getCallSites(frameCount)`
+## `util.getCallSites(frameCount, options)`
```

Comment:
The code reads to me that the first argument could be `options` as well. Should it be advocated since the `frameCount` is optional?

```suggestion
## `util.getCallSites([frameCountOrOptions, [options])`
* `frameCountOrOptions` {number} | {Object} Optional
* `options` Optional. Must be omitted if `frameCountOrOptions` is an {Object}.
```